packages/experiments/src/components/Text/Text.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import { IStyle } from '../../Styling';
-import { createComponent, IStyleProps, IViewProps } from './createComponent';
```

Comment:
Can you delete `src/components/Text/createComponent.tsx` now? Or not since it's used by `Stack`?

Reply:
I don't believe I can yet. `Stack` still uses `src/components/Text/createComponent.tsx`. The main difference between that and your new one is the statics property on `IComponentOptions`. Should that be another PR?